__controllers__/topics.controller.js

```diff
@@ -0,0 +1,7 @@
+const { selectTopics } = require("../__model__/topics.model");
+
+exports.getTopics = (req, res) => {
+  selectTopics().then((topics) => {
+    res.status(200).send({ topics });
+  });
```

Comment:
This is good so far but dont forget to add a catch block just in case something unexpected happens.